data/io.elementary.settings-daemon.gschema.xml

```diff
@@ -35,6 +35,18 @@
     </key>
   </schema>
 
+  <schema path="/io/elementary/settings-daemon/housekeeping/" id="io.elementary.settings-daemon.housekeeping">
+    <key type="b" name="cleanup-downloads-folder">
+      <default>false</default>
+      <summary>Whether the Downloads folder should be automatically cleaned up.</summary>
+      <description></description>
+    </key>
+    <key type="i" name="downloads-max-age-days">
```

Comment:
It looks like GSD just has one key `old-files-age` for both the trash and recent housekeeping options.

@cassidyjames do you think it would be better to have this individually configurable? Personally I think I'm fine with everything being kept for the same number of days

Reply:
@danrabbit I can see it both ways; on the one hand, one key is much simpler conceptually and easier to design for. On the other hand, I could see getting rid of trashed files after a very short amount of time while wanting the downloads folder to only clear out very old files—but I'm not sure if that's worth designing for.

Reply:
So have this listen to the already existing GSD key for how long to keep stuff, but have its own enable/disable key?

Reply:
It's probably worth having our own key since I imagine we'll want to move the other housekeeping features over here as well right?

I was mainly thinking the name of this key implies having an individual key for each thing to cleanup and wondering if we need that